script.js

```diff
@@ -6,19 +6,18 @@ const booksContainer = document.querySelector('.books-container');
 const getLocalBooks = localStorage.getItem('books');
 const localBooks = JSON.parse(getLocalBooks);
 let booksCollection = getLocalBooks ? localBooks : [];
-
-function addBook(title, author) {
-  const book = {
-    title,
-    author,
-    id: parseInt((Math.random() * 1000000000), 10),
-  };
-  booksCollection.push(book);
-  localStorage.setItem('books', JSON.stringify(booksCollection));
+class AddBook {
```

Comment:
- [ ] <b>REQUIRED:</b> `AddBook` should not be a class. it should be a function.